.ci/validation.cmd

```diff
@@ -32,6 +32,19 @@ if "%OPTION%" ==  "site_without_verify" (
   goto :END_CASE
 )
 
+if "%OPTION%" == "git_diff" (
+  for /f "delims=" %%a in ('git status ^| findstr /c:"Changes not staged"') do set output=%%a
+  if defined output (
+    echo Please clean up or update .gitattributes file.
+    echo Git status output:
+    echo Top 300 lines of diff:
```

Comment:
please remvoe `Git status output:` as line 40 already make it clear

Reply:
ok